pkg/aspectsupport/uber/logger.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package uber
+
+import (
+	"fmt"
+	"github.com/golang/glog"
+)
+
+type logger struct {
+	aspect string
+}
+
+func newLogger(aspect string) *logger {
+	return &logger{aspect: aspect}
+}
+
+func (l *logger) Infof(format string, args ...interface{}) {
+	glog.Infof(l.aspect + ":" + format, args...)
```

Comment:
The issue with this way of logging is that glog will always claim that `uber.go:31` is the logline.

We need to fork it.